.gitignore

```diff
@@ -48,14 +48,17 @@
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
-# Package Files #
+# Package Files
 *.jar
 *.war
 *.nar
 *.ear
 *.tar.gz
 *.rar
 
+# Eclipse generated files
```

Comment:
See below; the `.factorypath` file is added when formatting with eclipse-jdt. Actually it might also be reasonable to add [these lines](https://github.com/GreenBlitz/GB-Robot-Template/pull/189/files/9139b62b3a556a8a3719c752ca2cd20ca48085f1#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R156-R161) under this header, as they are also generated by eclipse-jdt, though not when formatting (so if you use the Java VSCode extension they will be generated, as it uses eclipse-jdt).